crates/sncast/src/starknet_commands/account/deploy.rs

```diff
@@ -98,17 +99,33 @@ async fn deploy_from_keystore(
     }
 
     let salt = account_data.get_salt_as_felt()?;
-    let class_hash = account_data.get_class_hash_as_felt()?;
 
     // TODO: Improve code for checking if address exists or remove this logic
```

Comment:
Please remove this comment because it should have been addressed in the refactoring PR. Although I still recommend improving this code in a separate issue.